From: "Ping Cheng" <pingc@wacom.com>
Subject: [PATCH] serial/8250_pnp.c: add new Wacom devices
Patch-mainline: 
References: bnc#544763

Add 10 serial Wacom Tablet PC devices. In fact, 
The last 3 bytes of WACF### are all reserved 
for Wacom serial OEMs.

Signed-off-by: Ping Cheng <pingc@wacom.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/serial/8250_pnp.c |   11 +++++++++++
 1 file changed, 11 insertions(+)

--- a/drivers/serial/8250_pnp.c
+++ b/drivers/serial/8250_pnp.c
@@ -337,6 +337,15 @@
 	{       "WACF00A",              0       },
 	{       "WACF00B",              0       },
 	{       "WACF00C",              0       },
+	{       "WACF00D",              0       },
+	{       "WACF00E",              0       },
+	{       "WACF00F",              0       },
+	{       "WACF010",              0       },
+	{       "WACF011",              0       },
+	{       "WACF012",              0       },
+	{       "WACF013",              0       },
+	{       "WACF014",              0       },
+	{       "WACF015",              0       },
 	/* Compaq touchscreen */
 	{       "FPI2002",              0 },
 	/* Fujitsu Stylistic touchscreens */
@@ -354,6 +363,8 @@
 	{	"FUJ02E5",		0	},
 	/* Fujitsu P-series tablet PC device */
 	{	"FUJ02E6",		0	},
+	/* Fujitsu Wacom Tablet PC device */
+	{	"FUJ02E7",		0	},
 	/*
 	 * LG C1 EXPRESS DUAL (C1-PB11A3) touch screen (actually a FUJ02E6 in
 	 * disguise)
